x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/constants.ts

```diff
@@ -136,6 +143,13 @@ export const CONFIG_INTRO_STEP1_HEADING = i18n.translate(
   }
 );
 
+export const CONFIG_INTRO_STEP1_BADGE = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.contentSource.configIntro.step1.badge',
```

Comment:
As per the i18n guidelines, can we change this to:
```suggestion
  'xpack.enterpriseSearch.workplaceSearch.contentSource.configIntro.step1.badgeLabel',
```

Reply:
RE: "id format" replied here: https://github.com/elastic/kibana/pull/112136#issuecomment-919985697